tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -846,6 +870,10 @@
     ),
 }
 
+if is_open('BZ:1887932'):
+    missing = '/katello/api/activation_keys/:activation_key_id/subscriptions'
+    API_PATHS['subscriptions'] = tuple([i for i in API_PATHS['subscriptions'] if i != missing])
```

Comment:
```suggestion
    API_PATHS['subscriptions'] = tuple([subscription for subscription in API_PATHS['subscriptions'] if subscription != missing])
```

Reply:
I used 'i' because flake8 wasn't happy with too long line and split into multiple lines, it looked awful

Reply:
I think we can use something meaningful instead of i.